libcxx/utils/ci/Dockerfile

```diff
@@ -132,6 +133,10 @@ RUN <<EOF
   sudo /tmp/llvm.sh $(($LLVM_HEAD_VERSION - 1)) all  # latest release
   sudo /tmp/llvm.sh $LLVM_HEAD_VERSION          all  # current ToT
   sudo apt-get install -y libomp5-$LLVM_HEAD_VERSION
+
+  # Install the latest git-clang-format
+  sudo ln -s $(which git-clang-format-$LLVM_HEAD_VERSION) /usr/bin/clang-format
```

Comment:
Did we change our policy? Shouldn't this be the latest stable version, `$(($LLVM_HEAD_VERSION - 1))`, instead?